frontend/src/features/auth/logoutModal.css

```
@@ -0,0 +1,133 @@
+ * style block on every page.
+ *
+ * Note: the `.auth-message-stack[hidden]` hide rule lives in tailwind.css (beside
+ * the render-blocking display:flex it overrides) so the closed modal never
+ * flashes full-screen before this async chunk loads.
+ */
+
+.logout-signout__card {
+	--so-panel-to: var(--bg-page);
+	--so-shadow: color-mix(in srgb, var(--bg-overlay-dark) 18%, transparent);
+	--so-ring: color-mix(in srgb, var(--text-muted) 28%, transparent);
+	align-items: center;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)